src/clickonce/launcher/HostFinder.cs

```diff
@@ -69,18 +110,26 @@ private static string GetProgramFilesFolder(bool is64bit = false)
         /// </summary>
         /// <param name="is64bit">If 64-bit bitness is required</param>
         /// <returns></returns>
-        private static string GetGlobalHost(bool is64bit = false)
+        private string GetGlobalHost(bool is64bit = false)
         {
             string folder = GetProgramFilesFolder(is64bit);
```

Comment:
I know this is not related to this PR exactly, but this is technically wrong. There's no guarantee that the runtime will be in program files. Ever since 3.1 we added the ability to install to custom "global" location and the location is written in registry. See this doc for the arch specific details: https://github.com/dotnet/designs/blob/main/accepted/2021/install-location-per-architecture.md
and this for the "custom global location" details: https://github.com/dotnet/designs/blob/main/accepted/2020/install-locations.md